packages/node-app/src/swagger.json

```
@@ -536,6 +536,86 @@
                     }
                 }
             }
+        },
+        "/login": {
+            "post": {
+                "tags": ["authentication"],
+                "description": "Login user",
+                "parameters": [
+                    {
+                      "name": "tokenType",
+                      "in": "query",
+                      "required": true
+                    },
+                    {
+                        "name": "accessToken",
+                        "in": "query",
+                        "required": true
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "User",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "object",
+                                    "example": {
+                                        "token": "auth-token"
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "401": {
+                        "description": "Invalid Authentication"
+                    },
+                    "5XX": {
+                        "description": "Unexpected error"
+                    }
+                }
+
+            }
+        },
+        "/me": {
+            "get": {
+                "tags": ["authentication"],
+                "description": "Get user",
+                "responses": {
+                    "201": {
+                        "description": "Logged in",
```

Just be in sync with the remaining documentation

```suggestion
                        "description": "OK",
```

---

OK is for a 200 status, this is 201 that also creates a user in the database if they don't exist there already so 'Logged in' is a more comprehensive description

---

Same as the previous comment, this is relative to /me endpoint, and not /login